integration/command.go

```diff
@@ -93,72 +93,109 @@ var cleanupInspektorGadget *command = &command{
 	cleanup: true,
 }
 
-// run runs the command on the given as parameter test.
-func (c *command) run(t *testing.T) {
-	if c.startAndStop {
-		if !c.started {
-			c.start(t)
+// createExecCmd returns an exec.Cmd for the command c.cmd. It will be
+// configured to store the stdout and stderr in command.stdout and
+// command.stderr so that we can use them on command.verifyOutput().
+func (c *command) createExecCmd() *exec.Cmd {
+	cmd := exec.Command("/bin/sh", "-c", c.cmd)
 
-			c.started = true
-		} else {
-			c.stop(t)
-		}
+	cmd.Stdout = &c.stdout
+	cmd.Stderr = &c.stderr
 
-		return
+	return cmd
+}
```

Comment:
I find this function a bit weird. It's acting on command but instead of setting `c.command` it return is and sometimes the callers needs to do `c.command = c.createExecCmd()`? 

Is it possible to modify this to set `c.command`? Also, I see it's invoked in different places, is it possible to invoke at the beginning of `run()` so start and stop have it ready?